lib/pipelines/code-pipeline.ts

```diff
@@ -65,7 +65,7 @@ export class CodePipeline {
                 // Install deps
                 'npm install',
                 // Install global CDK.
-                'npm install -g aws-cdk@1.104.0'
+                'npm install -g aws-cdk@1.113.0'
```

Comment:
@youngjeong46  have you validated cluster creation with 1.113?

Reply:
I validated and can confirm it works.